platform_umbrella/apps/home_base/test/home_base/customer_installs_test.exs

```diff
@@ -132,6 +133,10 @@ defmodule HomeBase.CustomerInstallsTest do
       assert_raise Ecto.NoResultsError, fn ->
         CustomerInstalls.get_installation!(installation.id)
       end
+
+      all_installations = Repo.all(Installation, with_deleted: true)
+      assert length(all_installations) == 1
+      assert installation.slug == all_installations |> List.first() |> Map.get(:slug)
```

Comment:
Please add a unit test that tests this soft delete of an installation more explicitly.

Reply:
Testing that `deleted_at` is set? I'm not sure I'm tracking otherwise.

Reply:
A test that inserts an installation, Then deletes it. Then tries to read to make sure that the soft deletes will work.
A test to make sure that we can read soft deletes. Insert an installation. delete it then read it with `with_deleted: true`.

You're wiring up soft deletes. We want to make sure that they don't get broken for any of the structs that use it.